README.md

```diff
@@ -1 +1,207 @@
+- Inline comments are available in the file for each of the source/filter/match blocks for easy reference for making any changes to the configuration.
+- Refer [this](https://docs.oracle.com/en/learn/oci_logging_analytics_fluentd/) to learn about each of the Logging Analytics Fluentd Output plugin configuration parameters.
+- *Note*: A generic source with time only parser is defined/configured for collecting all application pod logs from /var/log/containers/ out of the box. 
+          It is recommended to define and use a LogSource/LogParser at Logging Analytics for a given log type and then modify the configuration accordingly.
+          When adding a configuration (Source, Filter section) for any new container log, also exclude the log path from generic log collection, 
+            by adding the log path to *exclude_path* field in *in_tail_containerlogs* source block. This is to avoid the duplicate collection of logs through generic log collection.
+
+##### fluentd-daemonset.yaml
+
+- This file has all the necessary resources to deploy to run the Fluentd docker image as Daemonset.
+- Inline comments are available in the file describing each of the fields/sections. 
+- Make sure to replace the fields with actual values before deploying. 
+- At minimum, <IMAGE_URL>, <OCI_LOGGING_ANALYTICS_LOG_GROUP_ID>, <OCI_TENANCY_NAMESPACE> needs to be updated. 
+- It is recommended to update, <KUBERNETES_CLUSTER_OCID>,<KUBERNETES_CLUSTER_NAME> too tag all the logs with corresponding Kubernetes cluster at Logging Analytics. 
+
+##### secrets.yaml (Optional)
+
+- At present, InstancePrincipal and OCI Config File (UserPrincipal) based Auth/AuthZ are supported for Fluentd to talk to OCI Logging Analytics APIs. 
+- We recommend to use InstancePrincipal based AuthZ for OKE and all clusters which are running on OCI VMs and that is the default auth type configured. 
+- Applying this file is not required when using InstancePrincipal based auth type.
+- You need to modify this file to fill out the values under config section with appropriate values.
+
+##### Commands Reference
+
+Apply the yaml files in the sequence of configmap-docker.yaml(or configmap-cri.yaml), secrets.yaml (not required for default auth type) and fluentd-daemonset.yaml.
+
+```
+$ kubectl apply -f configmap-docker.yaml 
+configmap/oci-la-fluentd-logs-configmap created
+
+$ kubectl apply -f secrets.yaml 
+secret/oci-la-credentials-secret created
+
+$ kubectl apply -f fluentd-daemonset.yaml 
+serviceaccount/oci-la-fluentd-serviceaccount created
+clusterrole.rbac.authorization.k8s.io/oci-la-fluentd-logs-clusterrole created
+clusterrolebinding.rbac.authorization.k8s.io/oci-la-fluentd-logs-clusterrolebinding created
+daemonset.apps/oci-la-fluentd-daemonset created
+```
+
+You may use the following command to restart DaemonSet upon any applying any modifications to configmap or secrets to reflect the changes into the Fluentd.
```

Comment:
Use the following command to restart DaemonSet after applying any modifications to configmap or secrets, to reflect the changes into the Fluentd.